src/java.base/share/classes/java/nio/channels/FileChannel.java

```diff
@@ -146,7 +146,9 @@
  * operation first advances the position to the end of the file and then writes
  * the requested data.  Whether the advancement of the position and the writing
  * of the data are done in a single atomic operation is system-dependent and
- * therefore unspecified.
+ * therefore unspecified.  In this mode an invocation of the {@linkplain
+ * #write(ByteBuffer,long) absolute write} operation leads to unspecified
+ * behavior.
```

Comment:
I'm not sure about the phrase "absolute write" as we haven't used that before. I think it might be better to say something like "In this mode, the behavior of the method to write at a given position is also system-dependent".

Reply:
I was keying off the expression "In this mode each invocation of a relative write operation" earlier in the same paragraph and also in analogy to absolute and relative operations in `Buffer`. That said, I don't disagree with the comment.